### PR TITLE
Update getNodeDownloadUrl.ts

### PR DESCRIPTION
**Pull request by CodingRule**
Technically the same thing I did to PR #7241

### CODE REVIEW

**Comment by CodingRule** [comment]
If the comments are wrote wrongly please tell me!

File: apps/site/util/getNodeDownloadUrl.ts
Comment (avivkeller) [inline apps/site/util/getNodeDownloadUrl.ts:1]: Looking at the comparison, I think the switch statement is much cleaner.

Comment (CodingRule) [reply apps/site/util/getNodeDownloadUrl.ts:1]: Ah okay

Comment (CodingRule) [reply apps/site/util/getNodeDownloadUrl.ts:1]: Any good paths I could take into making this better than the switch cases ? (;

Comment (avivkeller) [reply apps/site/util/getNodeDownloadUrl.ts:1]: What's wrong with the switch statement?

Comment (CodingRule) [reply apps/site/util/getNodeDownloadUrl.ts:1]: Yeah true ig.
Thx anyway for review

Comment (avivkeller) [reply apps/site/util/getNodeDownloadUrl.ts:1]: No problem! Feel free to look at the issues tab for things that need to be fixed. "Help wanted" and "good first issue" are places to start.

Comment (AugustinMauroy) [reply apps/site/util/getNodeDownloadUrl.ts:1]: > Looking at the comparison, I think the switch statement is much cleaner.

Same feeling

**Review by avivkeller — CHANGES_REQUESTED** [review]
Based on my comment above, I dont think this should land in its current form.

**Review by ovflowd — COMMENTED** [review]
Not gonna lie, I actually was a fan of this PR, it looked way cleaner than the switch statement :)

Feel free to reopen it.

**Comment by avivkeller** [comment]
I'm still -1. I think that the switch statement is cleaner, but if you disagree, I won't block.

**Pull request closed without merging**